### ISSUE TITLE
Top and Bottom Pagination in sync

### ISSUE THREAD

**Issue by saumil-nagariya** [superRaytin/paginationjs#18]
Hello
Is it possible two pagination (TOP-BOTTOM) on a single page work in Sync? 

Actually, I want to add two pagination for same content. One is on the top of list and the other one is at the bottom of list. Is it possible? And obviously, both should run in Synchronization. If user clicks on top pagination on any page, same page should be selected in bottom pagination also.

**Comment by edmundo096** [comment]
I made a kind of simple change to support this when our init jQuery container reference to multiple elements #52

So you can have

```
<div class="pagination-container"></div>
<div class="pagination-container"></div>
```

and then

```
$('.pagination-container').pagination({...});
```

to work in sync.

On a simple test, they behave as expected, still it's not completely tested.

There is another PR #45 which tries to solve this same issue but seems the changes are broader.

**Comment by ramon-plusb** [comment]
Yeah this works but clicking on the page number results in a redirect to my startpage. Any workaround?
